src/scripts/pokemons/PokemonList.ts

```diff
@@ -14172,12 +14172,49 @@ const pokemonList: {
                 'speed': 34,
             },
         },
+        {
+            'id': -1,
+            'name': 'Mega-Venusaur',
+            'type': [PokemonType.Grass, PokemonType.Poison],
+            'eggCycles': 120,
+            'levelType': LevelType.mediumslow,
+            'exp': 306,
+            'catchRate': 45,
+            'base': {
+                'hitpoints': 80,
+                'attack': 100,
+                'specialAttack': 122,
+                'defense': 123,
+                'specialDefense': 120,
+                'speed': 80,
+            },
+        },
+        {
+            'id': -2,
+            'name': 'Mega-Charizard-X',
+            'type': [PokemonType.Fire, PokemonType.Dragon],
+            'eggCycles': 20,
+            'levelType': LevelType.mediumslow,
+            'exp': 306,
+            'catchRate': 45,
+            'base': {
+                'hitpoints': 78,
+                'attack': 130,
+                'specialAttack': 130,
+                'defense': 111,
+                'specialDefense': 85,
+                'speed': 100,
+            },
+
+
+        },
     ];
 
 
 // TODO move to its own initialize method that gets called on game start.
 pokemonList.forEach(p => {
-    p.attack = Math.max(...Object.values(p.base));
+
+    p.attack = Math.round(Math.max(Math.round(Math.max(1, Math.floor(Math.sqrt((2 * p.base.hitpoints)) * (2 * Math.round(Math.sqrt(p.base.attack * p.base.specialAttack) + Math.sqrt(p.base.speed)) * Math.sqrt(2 * Math.round(Math.sqrt(p.base.defense * p.base.specialDefense) + Math.sqrt(p.base.speed))))))) / 300));
```

Comment:
Could you please reduce the complexity of this formula by splitting it up into a few extra variables.
And there are a couple redundant rounding/max/floors in here too.
I think we should also make the minimum attack be 10 as 1 is just far too low of an attack amount.

If you need any help with these just @ me on the discord chat.

Reply:
done!